.github/workflows/npmpublish.yml

```diff
@@ -49,3 +56,43 @@ jobs:
       - run: ./scripts/deploy-docs.sh "${GITHUB_REF:1}"
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+      - name: Upload chart.js
+        id: upload-chart-js
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_AUTH_TOKEN }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: ./dist/chart.js
+          asset_name: chart.js
+          asset_content_type: text/javascript
+      - name: Upload chart.min.js
+        id: upload-chart-min-js
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_AUTH_TOKEN }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: ./dist/chart.min.js
+          asset_name: chart.min.js
+          asset_content_type: text/javascript
+      - name: Upload chart.esm.js
+        id: upload-chart-esm-js
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_AUTH_TOKEN }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: ./dist/chart.esm.js
+          asset_name: chart.esm.js
+          asset_content_type: text/javascript
+      - name: Upload helpers.esm.js
+        id: upload-helpers-esm-js
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_AUTH_TOKEN }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: ./dist/helpers.esm.js
```

Comment:
probably good idea to add the chunks too

Reply:
Maybe instead of a each individual file, what about creating a zip of the dist folder? That'd include the types too

Reply:
Doesn't `npm pack` already create a zip?

Reply:
It does, but I don't think we upload it anywhere